### ISSUE TITLE
Provide Dockerfile / Image

### ISSUE THREAD

**Issue by StefMa** [ioki-mobility/summaraizer-slack#6]
We want to make this code available with Docker.
For that we need a Dockerfile / Image.
We should use github packages to publish that image.

This depends on #5 

An example on how to do that can be found [here](https://github.com/zadjadr/prometheus-cve-exporter).
Check the `Dockerfile`, `.goreleaser.yml` and GitHub Action `release.yml` file.

**Comment by StefMa** [comment]
This is done.
See https://github.com/orgs/ioki-mobility/packages/container/package/summaraizer-slack
Follow-ups will get a new ticket
